framework/src/controller/channels/child_process_channel.ts

```diff
@@ -123,40 +155,52 @@ class ChildProcessChannel extends BaseChannel {
 		}
 	}
 
-	async invoke(actionName, params) {
-		const action =
-			typeof actionName === 'string'
-				? new Action(actionName, params, this.moduleAlias)
-				: actionName;
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	public async invoke(actionName: string, params?: object): Promise<any> {
+		const action = new Action(actionName, params, this.moduleAlias);
 
 		if (action.module === this.moduleAlias) {
-			return this.actions[action.name].handler(action);
+			// eslint-disable-next-line
+			// @ts-ignore
```

Comment:
It's because of the previous structure of the action object. The handler can be specified as an action property or directly as action definition. So it's optional here and causing this complaint. 

For sure that's need to be improved, but should we do it here in this PR or separate PR?